src/contracts/libraries/GPv2Order.sol

```diff
@@ -19,6 +19,8 @@ library GPv2Order {
         uint256 feeAmount;
         bytes32 kind;
         bool partiallyFillable;
+        bool useInternalSellTokenBalance;
+        bool useInternalBuyTokenBalance;
```

Comment:
I was wondering if this could maybe become part of the OrderKind enum to not blow up the order stride?

Reply:
It could, but I suspect that in the end it will end up being less gas efficient. Packing this stuff for call data definitely makes sense (which we currently do), but for execution memory, it makes a little less sense as the cost of access goes way up (as it requires additional math: `push offset mload push 2 shr push 1 and iszero` instead of just `push offset mload`). Likewise for writing this to memory, the cost of computing this field is probably more than the `mstore` (especially, since we allocate a single order and share that memory for a whole settlement, so memory allocation is relatively cheap for us).

Reply:
The other issue is that it would signing less ideal. RIght now we use a string `"sell"` or `"buy"` for the order kind, if we were to somehow mix this into that, then we couldn't any more.